packages/docusaurus-plugin-content-blog/package.json

```diff
@@ -3,15 +3,15 @@
   "version": "1.0.0",
   "description": "Blog content plugin for Docusaurus",
   "main": "index.js",
-  "files": [
-    "components"
-  ],
   "license": "MIT",
   "dependencies": {
     "@docusaurus/utils": "^1.0.0",
     "classnames": "^2.2.6",
     "fs-extra": "^7.0.1",
     "globby": "^9.1.0",
     "react": "^16.8.5"
+  },
+  "peerDependencies": {
+    "docusaurus": "^2.0.0"
```

Comment:
does this work now ? When doing yarn install at root ?

There is no docusaurus v2 package yet even in local

Reply:
Weird, it still works now. Will change it.